cmd/mount_unix.go

```diff
@@ -147,6 +147,10 @@ func mount_flags() []cli.Flag {
 			Name:  "update-fstab",
 			Usage: "add / update entry in /etc/fstab, will create a symlink at /sbin/mount.juicefs if not existing",
 		},
+		&cli.BoolFlag{
+			Name:  "grant-access",
+			Usage: "grant access to the /dev/fuse device (used in unprivileged containers)",
```

Comment:
We should try to do that when needed, don't add this option

Reply:
removed